certified-connectors/Telstra SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,720 @@
+                        "description": "value"
+                      },
+                      "fieldName": {
+                        "type": "string",
+                        "description": "fieldName"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "issue": {
+                        "type": "string",
+                        "description": "issue"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "issue": {
+                        "type": "string",
+                        "description": "issue"
+                      },
+                      "location": {
+                        "type": "string",
+                        "description": "location"
+                      },
+                      "value": {
+                        "type": "boolean",
+                        "description": "value"
+                      },
+                      "fieldName": {
+                        "type": "string",
+                        "description": "fieldName"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc. but not default. Kindly correct this in entire file.